package.json

```diff
@@ -29,6 +29,9 @@
     "babel-loader": "^6.2.5",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
+    "jquery": "^3.1.1",
```

Comment:
Not anymore, I planned to removed it yesterday, but couldn't remember what it was^^